[2024-06-03T03:55:20.450+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T03:55:20.487+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.load manual__2024-06-03T03:54:54.139707+00:00 [queued]>
[2024-06-03T03:55:20.496+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.load manual__2024-06-03T03:54:54.139707+00:00 [queued]>
[2024-06-03T03:55:20.497+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-03T03:55:20.510+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load> on 2024-06-03 03:54:54.139707+00:00
[2024-06-03T03:55:20.523+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1442) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-03T03:55:20.524+0000] {standard_task_runner.py:63} INFO - Started process 1444 to run task
[2024-06-03T03:55:20.523+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'load', 'manual__2024-06-03T03:54:54.139707+00:00', '--job-id', '250', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmp1lf6km0c']
[2024-06-03T03:55:20.525+0000] {standard_task_runner.py:91} INFO - Job 250: Subtask load
[2024-06-03T03:55:20.542+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-06-03T03:55:20.581+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_dag.load manual__2024-06-03T03:54:54.139707+00:00 [running]> on host aa989158b36e
[2024-06-03T03:55:20.675+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2024-06-03T03:54:54.139707+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-03T03:54:54.139707+00:00'
[2024-06-03T03:55:20.676+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T03:55:20.706+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/google/cloud/firestore_v1/base_collection.py:302 UserWarning: Detected filter using positional arguments. Prefer using the 'filter' keyword argument instead.
[2024-06-03T03:55:26.810+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-03T03:55:26.811+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T03:55:26.827+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=load, run_id=manual__2024-06-03T03:54:54.139707+00:00, execution_date=20240603T035454, start_date=20240603T035520, end_date=20240603T035526
[2024-06-03T03:55:26.897+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-03T03:55:26.915+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-03T03:55:26.917+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
